### PR TITLE
Missing `ai.djl.basicmodelzoo.basic.*` import in `train_your_first_model.ipynb`

### PR DESCRIPTION
**Pull request by mothguib**
## Description ##

Import of `ai.djl.basicmodelzoo.basic.*` was missing in tutorial `train_your_first_model.ipynb`, leading class `Mlp` to be unfindable.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- [x] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [x] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: jupyter/transfer_learning_on_cifar10.ipynb [hunk 1]
```diff
@@ -4,43 +4,30 @@
...
-    "\n",
-    "## Getting started\n",
-    "Load the Deep Java Libarary and its dependencies from Maven:"
+    "This tutorial requires the installation of the Java Jupyter Kernel. To install the kernel, see the [Jupyter README](https://github.com/awslabs/djl/blob/master/jupyter/README.md)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (frankfliu) [inline jupyter/transfer_learning_on_cifar10.ipynb]: Would you please clear output and save the jupyter notebook:

Kernel -> Restart & clear output

Comment (frankfliu) [reply jupyter/transfer_learning_on_cifar10.ipynb]: Please also squash all commits into one and force push your changes:

```
git rebase -i HEAD~8
```

**Comment by frankfliu** [comment]
@mothguib 

Thanks for your contribution, I rebased your PR and cleared jupyter notebook output. Since I don't have permission to push to your branch, I created a new PR: https://github.com/awslabs/djl/pull/32

File: jupyter/transfer_learning_on_cifar10.ipynb (hunk 1 above)
Comment (mothguib) [reply jupyter/transfer_learning_on_cifar10.ipynb]: Done

**Pull request closed without merging**